### DISCUSSION TITLE
SurrealDB with App Router

### DISCUSSION THREAD

**Discussion by wityan** [vercel/next.js#49436] [App Router]
I am currently experimenting with using SurrealDB inside a route Handler, but I am currently not able to make it work, I think its because of the Websocket Connection which is used by SurrealDB, is there someone who already got a working example or is there just no way to make it work? I think it would be quite Powerful to have Nextjs with SurrealDB

**Comment by henokmengistu** [comment]
I am having the same issue, what shall I do?

**Comment by Ngomana** [comment]
Please let me know if you found a work around solution.
